lib/widgets/message_list.dart

```diff
@@ -29,9 +29,8 @@ import 'theme.dart';
 class MessageListTheme extends ThemeExtension<MessageListTheme> {
```

Comment:
```
 theme: Replace dateSeparatorText and messageTimestamp with `labelTime`

Removed `dateSeparatorText` & `messageTimestamp` from
`MessageListTheme` and introduced `labelTime` in place of it
as per the design guidelines.
discussion: https://github.com/zulip/zulip-flutter/pull/1213#discussion_r1908359783

Fixes: #973
```

Because the commit message repeats the summary and the actual diff, we can remove most of it:

```
 theme: Replace dateSeparatorText and messageTimestamp with `labelTime`

discussion: https://github.com/zulip/zulip-flutter/pull/1213#discussion_r1908359783

Fixes: #973
```

We should remove the "Fixes" line because this is not that final commit that resolves the issue.

Let's also add a link to the Figma design.